src/xcode/ENA/ENA/Source/Services/Risk/Provider/Model/RiskConsumer.swift

```diff
@@ -33,7 +33,7 @@ final class RiskConsumer: NSObject {
 	var didCalculateRisk: ((Risk) -> Void) = { _ in }
 
 	/// Called when the risk calculation fails.
-	var didFailCalculateRisk: (RiskCalculationError) -> Void = { _ in }
+	var didFailCalculateRisk: (RiskProviderError) -> Void = { _ in }
```

Comment:
can we please get rid of this anti-pattern and `throw` errors!?

Reply:
Its just a renaming.

Reply:
throw errors / current used (anti) pattern might be a good discussion for dev camp?

Reply:
In the RiskConsumer we can use Result-Type. Don't see a possibility to use throws in RiskConsumer because its "async".

Reply:
We've discussed that. A potential fix for a future refactoring might be a `Result`.

Reply:
> We've discussed that. A potential fix for a future refactoring might be a `Result`.

Unfortunately I can't remember it - but maybe I was simply not there?